neuraxle/hyperparams/scipy_distributions.py

```diff
@@ -0,0 +1,1010 @@
+import math
+from abc import abstractmethod
+from typing import Optional, List, Any
+
+import numpy as np
+from scipy.integrate import quad
+from scipy.special import factorial
+from scipy.stats import rv_continuous, norm, rv_discrete, rv_histogram, truncnorm, randint
+
+from neuraxle.hyperparams.distributions import HyperparameterDistribution, WrappedHyperparameterDistributions
+
+def scipy_method(func):
+    def wrapper(*args, **kwargs):
+        self_in_args = args[0]
+        self_in_args._override_scipy_methods()
+        return func(*args, **kwargs)
+    return wrapper
+
+class ScipyDistributionWrapper(HyperparameterDistribution):
```

Comment:
So if I understand, our old distributions are still there, and we have these new ones ?

Reply:
exactly, I don't want to make any breaking changes. if the old ones become obselete we can mark them as obselete, but for now they are fine even with tpe !